gpu_bdb/bdb_tools/utils.py

```diff
@@ -65,7 +65,10 @@ def benchmark(func, *args, **kwargs):
     logging_info["elapsed_time_seconds"] = elapsed_time
     logging_info["function_name"] = name
     if compute_result:
-        import dask_cudf
+        if os.getenv("CPU_ONLY") == 'True':
+            import dask.dataframe as dask_cudf
+        else:
+            import dask_cudf
 
         if isinstance(result, dask_cudf.DataFrame):
```

Comment:
Below should just work , here dd is `import dask.dataframe as dd` . 

`dask_cudf.DataFrame` imports from `dask.DataFrame` so below just work. 

```suggestion
        if isinstance(result, dd.DataFrame):
```